fancy-explosions/BangazonSite
# Fixing soft delete for Products

## Type of change


# Testing Instructions

git fetch --all 
git checkout js-issue17
run it
make sure you can delete your products and then find me



# Checklist:
